pylink/tributaries/modulation.py

```diff
@@ -60,10 +59,19 @@ def __init__(self, name, tx_eff, rx_eff, esn0_db):
     Code("256APSK 32/45", 4.474530, 5.593162, 18.590000),
     Code("256APSK 11/15-L", 4.615190, 5.768987, 18.840000),
     Code("256APSK 3/4", 4.720684, 5.900855, 19.570000),
-    ]
+    Code("p2400 19.2", 0.3072, 0.384, 8.0),
```

Comment:
@polygnomial We should probably break these out into a separate array since they aren't a part of DVB-S2X.  I can worry about futzing with parts libraries later if necessary.